### ISSUE TITLE
Error: undefined method `name' for nil:NilClass when ensuring test devices are in Developer Portal

### ISSUE THREAD

**Issue by jordanekay** [bitrise-steplib/steps-ios-auto-provision#20]

**Comment by yonaskolb** [comment]
The stack trace:
```
...

Ensure test devices on Developer Portal
Error:
undefined method `name' for nil:NilClass
Stacktrace (for debugging):
/private/var/folders/90/5stft2v13fb_m_gv3c8x9nwc0000gn/T/bitrise616806697/step_src/lib/autoprovision/portal/device_client.rb:40:in `block in ensure_test_devices'
/private/var/folders/90/5stft2v13fb_m_gv3c8x9nwc0000gn/T/bitrise616806697/step_src/lib/autoprovision/portal/device_client.rb:16:in `each'
/private/var/folders/90/5stft2v13fb_m_gv3c8x9nwc0000gn/T/bitrise616806697/step_src/lib/autoprovision/portal/device_client.rb:16:in `ensure_test_devices'
/var/folders/90/5stft2v13fb_m_gv3c8x9nwc0000gn/T/bitrise616806697/step_src/step.rb:71:in `<main>'
```

**Comment by yonaskolb** [comment]
At first I thought this was a problem with the `TargetAttributes` as in #18, but this is a seperate error with `registered_test_device` https://github.com/bitrise-steplib/steps-ios-auto-provision/blob/master/lib/autoprovision/portal/device_client.rb#L40

**Comment by yonaskolb** [comment]
It looks like the error only occurs when the device isn't registered on the portal. The device does get successfully added to the portal though. This means that running again will be able to get through one more pending device

**Comment by godrei** [comment]
Hi @yonaskolb ,
we released a new version: [1.0.1](https://github.com/bitrise-steplib/steps-ios-auto-provision/releases/tag/1.0.1), which should fix this problem also.
I'll close this issue, but if you need more info/help just let us know and we'll reopen it!
